lib/ExpoRegistrar.php

```diff
@@ -115,4 +157,30 @@ private function isValidExpoPushToken(string $token)
     {
         return  substr($token, 0, 18) ===  "ExponentPushToken[" && substr($token, -1) === ']';
     }
+
+    /**
+     * Returns a driver class given a driver key.
+     *
+     * @param string $driver
+     * @return ExpoRepository
+     */
+    private function getRepository(string $driver)
```

Comment:
I'd remove this and instead rely on an interface and inject that interface via constructor.